client/src/components/Board.js

```diff
@@ -23,39 +34,59 @@ const useStyles = makeStyles((theme) => ({
                  : null,
       backgroundColor: props.status === "covered" ? "white" : props.status,
     },
+    fontSize: "1.5em",
+    fontWeight: "bold",
   }),
   board: {
     height: 500,
     width: "100%",
+    margin: theme.spacing(3, 0),
   },
   grid: {
     height: "inherit",
     width: "inherit",
-  }
+  },
+  spyHintNumberButton: {
+    maxWidth: "20px",
+    minWidth: "20px",
+  },
+  spyHintNumber: {
+    width: "30px",
+    userSelect: "none",
+  },
+  formButton: {
+    display: "flex",
+    margin: "24px auto 32px",
+    textTransform: "none",
+    backgroundColor: "#32BE72",
+    color: "white",
+    width: "120px",
+    height: "40px",
+  },
 }));
 
 const getRole = (player, gameState) => {
-  const {blueSpy, redSpy, blueGuessers, redGuessers} = gameState;
+  const { blueSpy, redSpy, blueGuessers, redGuessers } = gameState;
   if (player === blueSpy) return "blue spy";
   if (player === redSpy) return "red spy";
   if (blueGuessers.includes(player)) return "blue guesser";
   if (redGuessers.includes(player)) return "red guesser";
-}
+};
 
-const isSpy = role => role === "blue spy" || role === "red spy";
+const isSpy = (role) => role === "blue spy" || role === "red spy";
 
 const api = {
-  "nextMove": {
-    url: id => `/game/${id}/next-move`,
+  nextMove: {
+    url: (id) => `/game/${id}/next-move`,
```

Comment:
I think you might have different prettier settings than the rest of the team. We can double check this in standup